x/staketia/keeper/events.go

```diff
@@ -90,7 +90,6 @@ func EmitHaltZoneEvent(ctx sdk.Context, hostZone types.HostZone) {
 		sdk.NewEvent(
 			types.EventTypeHostZoneHalt,
 			sdk.NewAttribute(types.AttributeKeyHostZone, hostZone.ChainId),
-			sdk.NewAttribute(types.AttributeKeyRedemptionRate, hostZone.RedemptionRate.String()),
```

Comment:
Why not emit this? Import stakeibc and continue emitting.

Reply:
this function is no longer called from staketia